src/snowflake/cli/api/project/schemas/snowpark/callable.py

```diff
@@ -55,12 +45,13 @@ def convert_runtime(cls, runtime_input: Union[str, float]) -> str:
         return runtime_input
 
 
-class FunctionSchema(Callable):
+class FunctionSchema(_CallableBase, IdentifierModel(object_name="function")):  # type: ignore
```

Comment:
Why such approach? Because I wanted to generate object-specific help messages for database, schema and name fields. In this way the generated documentation (future) will be nicer.